pkg/remote/aws/route_table_supplier.go

```diff
@@ -38,9 +38,8 @@ func NewRouteTableSupplier(runner *parallel.ParallelRunner, client ec2iface.EC2A
 
 func (s RouteTableSupplier) Resources() ([]resource.Resource, error) {
 
-	retrievedRouteTables, err := listRouteTables(s.client)
+	retrievedRouteTables, err := listRouteTables(s.client, aws.AwsRouteTableResourceType)
```

Comment:
Why not handling error outside `listRouteTables` like in other suppliers ? It will avoid to pass the type everytime

Reply:
![image](https://user-images.githubusercontent.com/6154987/106125181-1f99f100-615c-11eb-839d-9b6f6b674171.png)

Reply:
this allows to factorize error trhowing for all the user of listRouteTables. Listbucket is a method of the client so it's different I don't think it should throw supplier errors.

Reply:
Ooops sorry wrong screenshot example my bad 🙏🏻 

![image](https://user-images.githubusercontent.com/6154987/106147581-5120b580-6178-11eb-98db-9bd2fbcc1401.png)

I mean it does not makes sense for me to factorize this error wraping as its bounded to the caller context. Thus you should updated `listRouteTables` method signature to add the type only for this error creation., we could `return nil, err` in `listRouteTables` and handle listing error in `Resources()` like in `EC2EbsSnapshotSupplier`.

Factorizing stuff just to have less line of codes is not the right thing to do for me in this context.

Reply:
ok lets close this as sometime we have two level of list method nesting, we need to make our suppliers more independent each others. So let's open a refactoring issue and close resolve this thread.